ext/CMakeLists.txt

```diff
@@ -22,38 +22,87 @@ endif()
 add_library(ext-tinyopt INTERFACE)
 target_include_directories(ext-tinyopt INTERFACE tinyopt/include)
 
+
```

Comment:
A few questions:
- is `add_sub_directory` insufficient?
- if so, why?
- should libgtest be installed? I'd prefer not to.
- do we need to re-set CXX and ilk?

Reply:
- The benchmark library uses the external project technique as well - by not using `add_sub_directory` we can limit the spill of the submodule content into our project. 
- libgtest (and google-benchmark for that matter) do not get installed
- haven't tried not setting it, but it shouldn't hurt

Reply:
ok, convinced me ;)